packages/client/src/views/OfficeHome/OfficeHome.test.tsx

```diff
@@ -230,11 +209,11 @@ describe('OfficeHome related tests', () => {
         app.find('#navigation_readyForReview').hostNodes().text()
       ).toContain('Ready for review3')
       expect(
-        app.find('#navigation_sentForUpdates').hostNodes().text()
-      ).toContain('Sent for updates4')
+        app.find('#navigation_requiresUpdateRegistrar').hostNodes().text()
+      ).toContain('Requires update4')
```

Comment:
what is update4?

Reply:
as we are trying to get the texts only, and there is no space on update and 4(though there is padding betwwen the containers), for this reason, there is no textual space between them.